.vscode/settings.json

```
@@ -36,4 +36,10 @@
     "quarto": true
   },
   "cSpell.language": "en,en-GB",
+  "cSpell.words": [
+    "Hmen",
+    "moveddown",
+    "movedup",
+    "nomoved"
+  ],
 }
```

```suggestion
}
```

We won't have these here for now, we need to eventually think of a place to put them. We don't want them in the settings file since we will eventually synch them with the template repo.